synthetics/data_source_agent.go

```diff
@@ -21,11 +20,11 @@ func dataSourceAgent() *schema.Resource {
 }
 
 func dataSourceAgentRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	tflog.Debug(ctx, fmt.Sprintf("Kentik API request - read agent with ID: %s\n", d.Get(idKey).(string)))
+	tflog.Debug(ctx, "Get synthetic Kentik API request", "ID", d.Get(idKey).(string))
 	resp, httpResp, err := m.(*kentikapi.Client).SyntheticsAdminServiceAPI.
 		AgentGet(ctx, d.Get(idKey).(string)).
 		Execute()
-	tflog.Debug(ctx, fmt.Sprintf("Kentik API response - read:\n%s\n", httpResp.Body))
+	tflog.Debug(ctx, "Get synthetic Kentik API response", resp)
```

Comment:
```suggestion
	tflog.Debug(ctx, "Get synthetic Kentik API response", "response", resp)
```
Could you show if the output of the log is correct? I think the Debug() API requires key-value pairs after the message.

In other places too.

Reply:
Done